step-release-vis/src/app/components/environments/environments.html

```diff
@@ -7,7 +7,7 @@
      [ngStyle]="{position: 'relative'}"
      (mouseup)="envsMouseUp()">
   <div class="env-header"
-       [ngStyle]="{height: getTimerangeHeight() + 'px'}">
+       [ngStyle]="{height: 2 * getTimerangeHeight() + 'px'}">
```

Comment:
why do we need more space here?

Reply:
so we have the dark mode button underneath the vision selector, everything is in the env-header

Reply:
IMO side by side, just grouped together is better, we do not want to lose too many space for this ;)

Reply:
done